app/modules/scorecard_api/service.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
```

Comment:
What's up with `app/modules` directory?
This is a namespaced class. It could live in `lib` (which would follow the pattern that `vets-api` uses) or `app/models`.
What is supposed to live here?
We have modules (`InstitutionBuilder`, `Archiver`) in `app/models`, so I'm unclear.

Reply:
The thought was to go ahead and make an `app/modules` directory so possibly in future we would reorganize the code base and move classes like `InstitutionBuilder`, `Archiver` to that directory

Reply:
Understood.  I'd consider using a different directory name though.  Maybe `utilities` instead of `modules`?
Ruby modules can be mixins, utility classes, namespacing tools.  It's a little vague.  Plus this particular file defines a `class` and not just a `module`.

Reply:
Renamed